Docs Update: Add Reown as one of the Dev Tools available on Rootstock

rohit-710 opened the pull request:
## Title
Adding Reown as one of the Dev Tools available on Rootstock for Account Abstraction and User Onboarding.

## Description
Reown provides fully open sourced and free to use SDKs for both Web3 apps and Web3 wallets. Reown also supports Rootstock, which means developers building a Web3 app on Rootstock can simply use Reown to power end-to-end wallet interactions on their app. 


## Testing

Tested locally.

## Checklist

- [x] I have read and understood the contributing guidelines.
- [x] I have followed the style guide and formatting guidelines.
- [x] I have added appropriate comments to explain the changes.
- [x] I have tested my changes thoroughly.

rohit-710 committed:
Create a user onboarding section

rohit-710 committed:
Create reown.md under user-onboarding

rohit-710 committed:
Update index.md

rohit-710 committed:
Create reown.md

rohit-710 committed:
Update index.md

owans commented on docs/05-dev-tools/account-abstraction/reown.md:
```diff
@@ -0,0 +1,24 @@
+tags: [reown, account abstraction, developer tools, rootstock, ethereum, dApps, wallets, solana, wallet connections, wallet infrastructure]
+---
+
+# Reown (prev. known as WalletConnect)
```
```suggestion
```

owans commented on docs/05-dev-tools/account-abstraction/reown.md:
```diff
@@ -0,0 +1,24 @@
+
+**[Reown](https://reown.com/?utm_source=rootstock&utm_medium=docs&utm_campaign=backlinks)** gives developers the tools to build user experiences that make digital ownership effortless, intuitive, and secure.
+
+### AppKit
```
```suggestion
## AppKit
```

owans commented on docs/05-dev-tools/user-onboarding/reown.md:
```diff
@@ -0,0 +1,29 @@
+tags: [reown, account abstraction, developer tools, rootstock, ethereum, dApps, wallets, solana, wallet connections, wallet infrastructure]
+---
+
+# Reown (prev. known as WalletConnect)
```
```suggestion
```

owans commented on docs/05-dev-tools/user-onboarding/reown.md:
```diff
@@ -0,0 +1,29 @@
+
+# Reown (prev. known as WalletConnect)
+
+**[Reown](https://reown.com/?utm_source=rootstock&utm_medium=docs&utm_campaign=backlinks)** gives developers the tools to build user experiences that make digital ownership effortless, intuitive, and secure.
```
```suggestion
**[Reown](https://reown.com/?utm_source=rootstock&utm_medium=docs&utm_campaign=backlinks)** provides developers with the tools to build user experiences that make digital ownership effortless, intuitive, and secure.
```

owans commented on docs/05-dev-tools/user-onboarding/reown.md:
```diff
@@ -0,0 +1,29 @@
+
+Reown has two major product offerings, they are, **AppKit** and **WalletKit**.
+
+### AppKit
```
```suggestion
## AppKit
```

owans commented on docs/05-dev-tools/user-onboarding/reown.md:
```diff
@@ -0,0 +1,29 @@
+
+AppKit is a powerful, free, and fully open-source SDK for developers looking to integrate wallet connections and other Web3 functionalities into their apps on any EVM and non-EVM chain. In just a few simple steps, you can provide your users with seamless wallet access, one-click authentication, social logins, and notifications—streamlining their experience while enabling advanced features like on ...
+
+### WalletKit
```
```suggestion
## WalletKit
```

owans commented on docs/05-dev-tools/user-onboarding/index.md:
```diff
@@ -0,0 +1,16 @@
+    title="Reown"
+    subtitle="User Onboarding"
+    color="green"
+    description="Reown (prev. known as WalletConnect) gives developers the tools to build user experiences that make digital ownership effortless, intuitive, and secure."
```
```suggestion
    description="Reown (prev. known as WalletConnect) provides developers with the tools to build user experiences that make digital ownership effortless, intuitive, and secure."
```

owans commented on docs/05-dev-tools/user-onboarding/reown.md:
```diff
@@ -0,0 +1,29 @@
+sidebar_label: Reown
+title: User Onboarding with Reown
+description: "Reown gives developers the tools to build user experiences that make digital ownership effortless, intuitive, and secure." 
+tags: [reown, account abstraction, developer tools, rootstock, ethereum, dApps, wallets, solana, wallet connections, wallet infrastructure]
```
```suggestion
tags: [reown, account abstraction, developer tools, rootstock, ethereum, dApps, wallets, wallet connections, wallet infrastructure]
```

owans commented on docs/05-dev-tools/account-abstraction/reown.md:
```diff
@@ -0,0 +1,24 @@
+sidebar_label: Reown
+title: Account Abstraction using Reown
+description: "Reown gives developers the tools to build user experiences that make digital ownership effortless, intuitive, and secure." 
+tags: [reown, account abstraction, developer tools, rootstock, ethereum, dApps, wallets, solana, wallet connections, wallet infrastructure]
```
```suggestion
tags: [reown, account abstraction, developer tools, rootstock, ethereum, dApps, wallets, wallet connections, wallet infrastructure]
```

owans commented:
Hi @rohit-710 Thank you for contributing to the docs and adding Reown DevTool. Please attend to the comments and I will merge.

rohit-710 committed:
Update docs/05-dev-tools/account-abstraction/reown.md

Co-authored-by: Owanate Amachree <amachreeowanate@gmail.com>

rohit-710 committed:
Update docs/05-dev-tools/account-abstraction/reown.md

Co-authored-by: Owanate Amachree <amachreeowanate@gmail.com>

rohit-710 committed:
Update docs/05-dev-tools/user-onboarding/reown.md

Co-authored-by: Owanate Amachree <amachreeowanate@gmail.com>

rohit-710 committed:
Update docs/05-dev-tools/user-onboarding/reown.md

Co-authored-by: Owanate Amachree <amachreeowanate@gmail.com>

rohit-710 committed:
Update docs/05-dev-tools/user-onboarding/reown.md

Co-authored-by: Owanate Amachree <amachreeowanate@gmail.com>

rohit-710 committed:
Update docs/05-dev-tools/user-onboarding/reown.md

Co-authored-by: Owanate Amachree <amachreeowanate@gmail.com>

rohit-710 committed:
Update docs/05-dev-tools/user-onboarding/index.md

Co-authored-by: Owanate Amachree <amachreeowanate@gmail.com>

rohit-710 committed:
Update docs/05-dev-tools/user-onboarding/reown.md

Co-authored-by: Owanate Amachree <amachreeowanate@gmail.com>

rohit-710 committed:
Update docs/05-dev-tools/account-abstraction/reown.md

Co-authored-by: Owanate Amachree <amachreeowanate@gmail.com>

rohit-710 commented:
> Hi @rohit-710 Thank you for contributing to the docs and adding Reown DevTool. Please attend to the comments and I will merge.

Hey @owans, thank you review the PR. I just addressed all of your comments. Let me know if it looks good to go now 🙏

owans commented on docs/05-dev-tools/account-abstraction/index.md:
```diff
@@ -59,14 +59,14 @@ The Rootstock network is innovating in the account abstraction space, here are s
-    linkHref="/dev-tools/dev-environments/gelato/"
-    linkTitle="Build with Gelato"
-  /> -->
+    description="Reown (prev. known as WalletConnect) gives developers the tools to build user experiences that make digital ownership effortless, intuitive, and secure."
```
```suggestion
    description="Reown (prev. known as WalletConnect) gives developers the tools to build user experiences that make digital ownership effortless, intuitive, and secure."
```
```suggestion
    description="Reown (prev. known as WalletConnect) provides developers with the tools to build user experiences that make digital ownership effortless, intuitive, and secure."
```

owans commented:
> > Hi @rohit-710 Thank you for contributing to the docs and adding Reown DevTool. Please attend to the comments and I will merge.
> 
> Hey @owans, thank you review the PR. I just addressed all of your comments. Let me know if it looks good to go now 🙏

Thanks @rohit-710 Just one minor comment remaining.

This contribution also qualifies for the ongoing Rootstock Hacktivator Program: https://dev.rootstock.io/resources/contribute/hacktivator-program/

You can submit this PR here: https://forms.gle/aF9pFMfRyoygfzJWA

Once again. Thank you for contributing!

rohit-710 committed:
Update docs/05-dev-tools/account-abstraction/index.md

Co-authored-by: Owanate Amachree <amachreeowanate@gmail.com>

rohit-710 commented:
> > > Hi @rohit-710 Thank you for contributing to the docs and adding Reown DevTool. Please attend to the comments and I will merge.
> > 
> > 
> > Hey @owans, thank you review the PR. I just addressed all of your comments. Let me know if it looks good to go now 🙏
> 
> Thanks @rohit-710 Just one minor comment remaining.
> 
> This contribution also qualifies for the ongoing Rootstock Hacktivator Program: https://dev.rootstock.io/resources/contribute/hacktivator-program/
> 
> You can submit this PR here: https://forms.gle/aF9pFMfRyoygfzJWA
> 
> Once again. Thank you for contributing!

Awesome, thanks for letting me know! I missed that comment, just made a commit for the same.

owans reviewed (approved):
LGTM!
